src/org/labkey/test/components/glassLibrary/grids/EditableGrid.java

```diff
@@ -297,20 +340,129 @@ public void setCellValue(int row, String columnName, Object value)
             // the cell's new value will be the old value plus the new value and the cursor may not be placed at the end
             // of the existing value so the new value should exist some where in the cell text value not necessarily
             // at the end of it.
-            getWrapper().waitFor(() -> gridCell.getText().contains(value.toString()),
+            WebDriverWrapper.waitFor(() -> gridCell.getText().contains(value.toString()),
                     "Value entered into inputCell '" + value + "' did not appear in grid cell.", WAIT_FOR_JAVASCRIPT);
 
         }
 
     }
 
     /**
-     * pastes delimited text to the grid, from a single target.  The component is clever enough to target
+     * For a given row get the value in the given column.
+     *
+     * @param row The row index (0 based).
+     * @param columnName The name of the column to get the value for.
+     * @return The string value of the {@link WebElement} that is the cell.
+     */
+    public String getCellValue(int row, String columnName)
+    {
+        // Get a reference to the cell.
+        WebElement gridCell = getCell(row, columnName);
+
+        return gridCell.getText().trim();
+    }
+
+    /**
+     * Get the displayed pick-list for the given grid cell (td).
+     *
+     * @param gridCell The td element that contains the list.
+     * @return A list of the items in the list.
+     */
+    private List<String> getPickList(WebElement gridCell)
+    {
+        WebElement divLookup = Locators.lookupMenu.findWhenNeeded(gridCell);
+
+        // Wait for the picklist to show.
+        WebDriverWrapper.waitFor(divLookup::isDisplayed,
+                "The picklist for the cell did not appear in time.", 5_000);
+
+        List<WebElement> items = Locators.lookupItem.findElements(divLookup);
+        return getWrapper().getTexts(items);
+    }
+
+    /**
+     * Dismiss the pick-list for the cell in the grid.
+     *
+     * @param row A 0 based index for the row that the cell is on.
+     * @param columnName The name of the column for the cell.
+     * @return A reference to this EditableGrid.
+     */
+    public EditableGrid dismissPickList(int row, String columnName)
+    {
+        WebElement gridCell = getCell(row, columnName);
+        Actions builder = new Actions(getDriver());
+        builder.sendKeys(gridCell, Keys.ESCAPE).build().perform();
+
+        return this;
+    }
```

Comment:
Is there a way to make this work without targeting a specific cell? Just to simplify it, since there can only be one pick-list open (I presume). Maybe something like:
```suggestion
    public EditableGrid dismissPickList()
    {
        Actions builder = new Actions(getDriver());
        builder.sendKeys(getComponentElement(), Keys.ESCAPE).build().perform();

        return this;
    }
```

Reply:
Changed.